app/javascript/features/form/components/DialogContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  Button
+} from "@material-ui/core";
+import React from "react";
+
+type OpenDialog = {
+  isOpenSuccessDialog: boolean;
+  setIsOpenSuccessDialog: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const DialogContainer = (props: OpenDialog) => {
```

Comment:
```suggestion
const SuccessDialog = (props: OpenDialog) => {
```
名前はSuccessDialogとかのほうがよさそうに思います